extensions/scarb-snforge-test-collector/src/metadata.rs

```diff
@@ -122,7 +123,10 @@ impl CompilationUnit<'_> {
                         } else {
                             Edition::default()
                         },
-                        cfg_set: None,
+                        cfg_set: component.cfg.clone().map(|cfg| CfgSet::from_iter(cfg.iter().map(|cfg|             serde_json::to_value(cfg)
```

Comment:
This oneliner is really horrible to read 😬

Reply:
Refactored